src/i_video.c

```diff
@@ -1560,16 +1565,6 @@ static void I_InitGraphicsMode(void)
 
    I_GetWindowPosition(&window_x, &window_y, v_w, v_h);
 
-#ifdef _WIN32
-   // [JN] Windows 11 idiocy. Indicate that window using OpenGL mode (while it's
-   // a Direct3D in fact), so SDL texture will not be freezed upon vsync
-   // toggling.
-   if (I_CheckWindows11())
```

Comment:
Can we remove this function entirely?

Reply:
Done in https://github.com/fabiangreffrath/woof/pull/977/commits/01e685758f0babfc1d4bdd25958de1e4a86d0c15 Is it OK to merge? It works surprisingly well, and in exclusive full screen mode too.